code/modules/arcade/mob_hunt/game_server.dm

```diff
@@ -0,0 +1,164 @@
+var/global/datum/controller/process/mob_hunt/mob_hunt_server
```

Comment:
I'm going to be pedantic and grump about there being a controller here as opposed to with all the others.